### DISCUSSION TITLE
[SUGGESTION] VLAN tag

### DISCUSSION THREAD

**Discussion by alex-ballos** [snappysnappydog/Simple-IP-Config#189] [Ideas]
It would be awesome if i could specify VLAN id
like this
"Set-NetAdapter -Name "Ethernet" -VlanID 178"

**Comment by KurtisLiggett** [comment]
Converting this to discussion - Idea.

**Comment by KurtisLiggett** [comment]
I don't have any plans to add VLAN related settings for now.
But I will leave this open to see if there is any other interest.

**Comment by JonnyTech** [comment]
Is it possible to set vlans in standard windows without simple-ip-config anyway? I do not see an option..
